metadata_service/proxy/neo4j_proxy.py

```diff
@@ -752,33 +751,28 @@ def get_table_by_user_relation(self, *, user_email: str, relation_type: UserReso
         :return:
         """
         relation, _ = self._get_relation_by_type(relation_type)
-        # relationship can't be parameterized
-        query_key = 'key: "{user_id}"'.format(user_id=user_email)
 
         query = textwrap.dedent("""
-        MATCH (user:User {{{key}}})-[:{relation}]->(tbl:Table)
-        RETURN COLLECT(DISTINCT tbl) as table_records
-        """).format(key=query_key,
-                    relation=relation)
+MATCH (user:User {{key: $query_key}})-[:{relation}]->(tbl:Table)-[:TABLE_OF]->
```

Comment:
the change looks good, but is there a way to extend this query to support the frequent use use case?
Here is my initial query I have in mind
```
MATCH (user:User {key: 'tfeng@lyft.com'})-[r:READ]->(tbl:Table)
WITH tbl.key as table_key, r.read_count as total_reads, r.published_tag as published_tag
where published_tag IS NOT NULL
RETURN  table_key as table_records, total_reads, published_tag  order by published_tag desc, total_reads desc limit 50;
```

Reply:
Mmm. We might can squeeze it to make it shareable, but I am afraid it would make it complicated. How about separating it out?

Reply:
Yeah. I think separate it out will be fine. Do you think you could combine that into this pr so that we could ship the change together? Thanks :)

Reply:
sync offline. Let's push the change and add a todo on the query .

Reply:
@feng-tao Just added `get_frequently_used_tables` method for frequently used table. :)